src/components/pages/course-form/CourseFormContainer.jsx

```diff
@@ -1,11 +1,102 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Form, Input, Space, Button } from 'antd';
+import { ModuleForm, ModuleCard } from '../module-form';
+import styled from 'styled-components';
+
+const StyledSpace = styled(Space)`
+  &&& {
+    width: 100%;
+  }
+`;
+
+// CourseFormContainer — No specific parent, props isn't coming down quite yet
+export default props => {
+  const [form] = Form.useForm();
+  const [modalVisible, setModalVisible] = useState(false);
+
+  const showModuleModal = () => setModalVisible(true);
+  const hideModuleModal = () => setModalVisible(false);
+
+  const onFinish = values => {
+    // we will do something different here once
+    // we're ready to hit endpoints!!
+    console.log({ values });
+  };
+
+  const onModuleAdd = newModule => {
+    const existingModules = form.getFieldValue('modules') || [];
+    form.setFieldsValue({
+      modules: [...existingModules, newModule],
+    });
+    setModalVisible(false);
+  };
 
-const CourseFormContainer = props => {
   return (
-    <>
-      <h1>CourseFormContainer</h1>
-    </>
+    <StyledSpace direction="vertical" align="center">
+      <h1>Course Form</h1>
+      <Form
+        form={form}
+        name="courseForm"
+        onFinish={onFinish}
+        autoComplete="off"
+        layout="vertical"
+      >
+        <Form.Item
+          name="coursename"
+          label="Course Name"
+          rules={[{ required: true, message: 'Missing Course Name' }]}
```

Comment:
I think this is a discussion we can have as a team, but we should consider how we want to convey these sorts of errors to the end user. Perhaps something more friendly and along the lines of 'Please add a course name' would be good. Let's leave it this way for now, but I think it's a topic for later discussion!